isis/src/mro/apps/hical/hical.xml

```diff
@@ -9,70 +9,70 @@
 
   <description>
      <p>
-        <b>hical</b> appiles radiometric calibration correction to HiRISE 
+        <b>hical</b> appiles radiometric calibration correction to HiRISE
         images.  This particular version is deemed experimental, thus the beta
```

Comment:
Can we get rid of the "beta designation" language here?  I think this must be an echo for when "hical" was copied over from "hicalbeta" back in the before-time.  Also, hicalbeta is probably hopelessly out-of-date, but that's probably a different PR.

Reply:
We've kept hical and hicalbeta basically the same, but no longer copy hicalbeta to hical. I don't know about removing the hicalbeta language here. What do you think about a separate PR to remove hicalbeta and the language?